python/tests/operations/random_like.py

```diff
@@ -0,0 +1,145 @@
+import os
+import unittest
+
+import numpy as np
+
+import equistore
+
+
+DATA_ROOT = os.path.join(os.path.dirname(__file__), "..", "data")
+
+
+class TestRandomLike(unittest.TestCase):
```

Comment:
i know that it will be a pain but we are moving to pytest, and any new addition to the code have to have the test in pytest not unittest